specs/core/0_beacon-chain.md

```diff
@@ -2428,11 +2428,16 @@ def process_transfer(state: BeaconState, transfer: Transfer) -> None:
     )
     # A transfer is valid in only one slot
     assert state.slot == transfer.slot
-    # Only withdrawn or not-yet-deposited accounts can transfer
+    # Only withdrawn or not-yet-activated accounts can transfer
     assert (
         get_current_epoch(state) >= state.validator_registry[transfer.sender].withdrawable_epoch or
         state.validator_registry[transfer.sender].activation_epoch == FAR_FUTURE_EPOCH
     )
+    # Verify that the validator recieving the transfer is either withdrawn or not-yet-activated
+    assert (
```

Comment:
Why not `assert not is_active_validator(state.validator_registry[transfer.recipient])`?